resources/VERSION

```diff
@@ -0,0 +1 @@
+v0.2.2-59-ga88853a
```

Comment:
I read the commits individually, because I thought they were independent.

In the future you could try "fixup" commits and see if that helps. That will create a new commit with the intention to modify an existing commit. When a review is done you can do an interactive rebase with the `--autosquash` flag which will automatically set the correct commits to "squash". See this [1] for more information. Using fixup commits might actually help as well on GitHub because I think that it's more likely that the comments will persist.

[1] https://robots.thoughtbot.com/autosquashing-git-commits